src/SignalR/clients/csharp/Http.Connections.Client/src/Internal/ITransport.cs

```diff
@@ -3,14 +3,15 @@
 
 using System;
 using System.IO.Pipelines;
+using System.Threading;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Connections;
 
 namespace Microsoft.AspNetCore.Http.Connections.Client.Internal
 {
     public interface ITransport : IDuplexPipe
     {
-        Task StartAsync(Uri url, TransferFormat transferFormat);
+        Task StartAsync(Uri url, TransferFormat transferFormat, CancellationToken cancellationToken);
```

Comment:
Make this a default parameter, and remove CancellationToken.None from all the argument passing. This is something that we do in our other APIs.

Reply:
Do you also want me to use the token from StartAsync() for webSockets StartReceiving() and StartSending()?

Reply:
Nope.